README

```diff
@@ -75,7 +75,7 @@ Top level documentation ($CTSMROOT):
 README ------------------- This file
 README.md ---------------- File that displays on github under https::/github.com/ESCOMP/CTSM.git
 README.rst --------------- File that displays under the project in github
-README_EXTERNALS.rst ----- Information on how to work with manage_externals for CTSM (deprecated)
+README_EXTERNALS.rst ----- Information on how to work with git-fleximod for CTSM
```

Comment:
I see below you removed it. I suggest creating a new file that gives details into how to do the same things with git fleximod.

Reply:
To me it makes sense to rename the file to README_GITSUBMODULES.rst if we also rename externals to submodules. Is this the plan?

Alternatively, we could call it README_GIT-FLEXIMOD.rst to avoid having too many names for the same thing.